<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KuttyPy Interactive Console</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="3" column="0" rowspan="2" colspan="2">
     <widget class="QTextBrowser" name="log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPanel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>GM Counter</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="examples">
       <attribute name="title">
        <string>Code Snippets</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/control/play.svg</normaloff>:/control/play.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="exampleList"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QPlainTextEdit" name="userCode">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="plainText">
           <string/>
          </property>
          <property name="tabStopWidth">
           <number>20</number>
          </property>
          <property name="backgroundVisible">
           <bool>false</bool>
          </property>
          <property name="centerOnScroll">
           <bool>false</bool>
          </property>
          <property name="class" stdset="0">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/control/stop.svg</normaloff>:/control/stop.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="enableLog">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="clearLog">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Auto-Clear</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Parameter View</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="controldock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="dockLayout"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>runCode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabs</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>tabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>abort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exampleList</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>loadExample(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>runCode()</slot>
  <slot>tabChanged(int)</slot>
  <slot>abort()</slot>
  <slot>loadExample(QString)</slot>
  <slot>compile()</slot>
  <slot>upload()</slot>
  <slot>saveCFile()</slot>
  <slot>openCFile()</slot>
  <slot>newRegister()</slot>
  <slot>userRegistersAutoRefresh(bool)</slot>
 </slots>
</ui>
